deps/rabbitmq_management_agent/src/rabbit_mgmt_format.erl

```diff
@@ -78,6 +78,12 @@ format_queue_stats({disk_reads, _}) ->
     [];
 format_queue_stats({disk_writes, _}) ->
     [];
+format_queue_stats({members, Value}) ->
+    [{members, lists:sort(Value)}];
```

Comment:
Should this be `lists:usort/1`? `lists:sort/1` does not eliminate duplicates but `lists:usort/1` does.

Reply:
Perhaps I misunderstood the intent. If sorting members addresses the root cause. This can be enough. In theory, there should be no duplicate members reported by a QQ.